common/test/acceptance/pages/studio/users.py

```diff
@@ -78,6 +80,48 @@ def click_submit_new_user_form(self):
         self.q(css='.form-create.create-user .action-primary').click()
         wait_for_ajax_or_reload(self.browser)
 
+    def get_user(self, email):
+        """ Gets user wrapper by email """
+        target_users = [user for user in self.users if user.email == email]
+        assert len(target_users) == 1
```

Comment:
Pages methods shouldn't do asserts. In this sort of method, we just return the first element.